### PR TITLE
[MIG][14.0] product brand purchase report

### PR DESCRIPTION
**Pull request by ilyasProgrammer**
Migrated product brand purchase report
Renamed to product brand purchase 
Added brand menu under purchase config.

### CODE REVIEW

**Review by francesco-ooops — APPROVED** [review]
Functional review ok! 

This module allows users to manage brands in purchase app.

**Review by geomer198 — APPROVED** [review]
Code Review ok!

**Comment by francesco-ooops** [comment]
@max3903 @sbejaoui could this work?

File: product_brand_purchase/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#. Go to *Purchase > Reporting > Purchase*.
+#. You can now group by brand.
```
Comment (francesco-ooops) [inline product_brand_purchase/readme/USAGE.rst]: #. "Product Brands" menu is now accessible also under *Purchase > Configuration > Products*

File: product_brand_purchase/views/product_brand_view.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+    <menuitem
+        id="menu_product_in_config_purchase_product_brand"
```
Comment (francesco-ooops) [inline product_brand_purchase/views/product_brand_view.xml:5]: name="Product Brands"

**Review by francesco-ooops — CHANGES_REQUESTED** [review]
@ilyasProgrammer

**Comment by ilyasProgrammer** [comment]
@francesco-ooops updated

**Comment by francesco-ooops** [comment]
@OCA/brand-maintainers is this good for merge?

**Comment by francesco-ooops** [comment]
@sbejaoui if you have a moment to check this, thanks!

**Review by sbejaoui — APPROVED** [review]
LGTM, code review

**Pull request merged**